packages/list-reusable-blocks/src/utils/export.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { pick, kebabCase } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { download } from './file';
+
+/**
+ * Export a reusable block as a JSON file.
+ *
+ * @param {number} id
+ */
+async function exportReusableBlock( id ) {
+	const postType = await apiFetch( { path: `/wp/v2/types/wp_block` } );
+	const reusableBlock = await apiFetch( { path: `/wp/v2/${ postType.rest_base }/${ id }` } );
+	const fileContent = JSON.stringify( {
+		__file: 'wp_block',
```

Comment:
yes, this is just a marker. My idea is that this will remove any ambiguity while trying to auto-import this file (like if we implement drag and drop to the editor). We could detect that the imported file has a "title" and "content" strings (JSON Schema) but this seems too common, I prefer as always explicitness over implicitness. What if we add another exported file with the same format that should imported differently? (like in a File block or something).

I also considered adding a `version` field if ever we change the format but I decided against because we can think of a missing version as `version: 0`

Reply:
Another thought: Do we ever intend to support batch import / export? Would this file contain multiple at some point? Will this format support that as a future enhancement?

Reply:
Explicitness is fine, just seems a format arbitrarily decided upon.